src/ios/CDVOrientation.m

```diff
@@ -28,6 +28,74 @@ @interface CDVOrientation () {}
 
 @implementation CDVOrientation
 
+-(void)handleAboveEqualIos16WithOrientationMask:(NSInteger) orientationMask viewController: (CDVViewController*) vc result:(NSMutableArray*) result selector:(SEL) selector
+{
+    if(@available(iOS 16.0, *)) {
```

Comment:
Can you please confirm, if this iOS 16 conditional wrapper can be removed?

I believe the method is called only on line 127 and it is already wrapped by an iOS 16 conditional check (Line 126). If so we can remove the extra if conditional check and de-indent the lines.

Other thing this.. I think everything looks good now and cleaner.

Reply:
Actually can you try and replace the wrapper with this instead?

```objective-c
#if __IPHONE_OS_VERSION_MAX_ALLOWED > __IPHONE_15_5
// Code Here
#endif
```

Just to see if we can fix the CI.

I ran the CI and it failed, but this is because the earlier iOS versions are built with a version of XCode that doesn't understand the newer identifiers.

E.g. error message:
> error: use of undeclared identifier 'UIWindowSceneGeometryPreferencesIOS'

Reply:
it doesn't look like I can replace it, because if I do then it fails to compile on Xcode14 (because on Xcode14 max allow is iOS16) - but I think I can use a combination of the two directives.

the wrapper on line 33 is there to stop Xcode complaining with warnings, I am trying to see if I can add a pragma just for that function but I dont seem to be able to find something sensible.

anyway new PR coming up.